### ISSUE TITLE
Dev mode halting terminal process

### ISSUE THREAD

**Issue by mchatzis** [anomalyco/sst#4972]
Version >= 3.2.51 struggles to kill the dev process when either `x` or `Ctrl + C` is pressed, either halting or severly delaying.

command: npx sst@3.2.51 dev
platform: MacOS M1, z-shell

Edit:
Current `sst.config.ts`

```
/// <reference path="./.sst/platform/config.d.ts" />

export default $config({
  app(input) {
    return {
      name: "appName",
      removal: input?.stage === "production" ? "retain" : "remove",
      home: "aws",
    };
  },
  async run() {
    const app = new sst.aws.Nextjs("MyWeb", {
      path: "pathToNextjsApp"
    });

    return {
      app: app.url,
    };
  },
});
```

**Comment by hayedewit** [comment]
Same issue

**Comment by thdxr** [comment]
can you send me the last few lines of your `.sst/log/sst.log`

**Comment by thdxr** [comment]
nevermind able to recreate on mac os let me try

**Comment by thdxr** [comment]
fixed in v3.2.55

**Comment by gustavgenberg** [comment]
https://discord.com/channels/983865673656705025/1299379633191391315
